modules/modify-unavailable-workers.adoc

```diff
@@ -30,3 +30,12 @@ spec:
 When setting the value, consider the number of worker nodes that can be
 unavailable without affecting the applications running on the cluster.
 ====
+
+[IMPORTANT]
+====
+Care must be exercised when modifying the `KubeletConfig` as improper 
```

Comment:
Should we be bolder here and say 'Not Supported'? 
cc: @sjenning

Reply:
@rvanderp3 I am not clear on what this procedure does related to your note. 

Does this procedure modify the `KubeletConfig` for workers? Or, does this mean, when a user creates or edits a `KubeletConfig`? 
If the latter, should we put the note in those sections that describe creating or modifying a `KubeletConfig` CR? 
@sferich888 ^^

Reply:
@sferich888 @sjenning Is creating and modifying a `KubeletConfig` CR not supported?

Reply:
> @rvanderp3 I am not clear on what this procedure does related to your note.
> 
> Does this procedure modify the `KubeletConfig` for workers? Or, does this mean, when a user creates or edits a `KubeletConfig`?
> If the latter, should we put the note in those sections that describe creating or modifying a `KubeletConfig` CR?
> @sferich888 ^^

@mburke5678 Yes, it relates to the latter.

Reply:
@sferich888 no, but this is a large gun with which one can shoot themselves in the foot.  The more validation we can add to the MCO over time, the better.  fyi @rphillips